### PR TITLE
footer implementation

### LINKED ISSUE #62: Footer implementation
**Is your feature request related to a problem? Please describe.**
Adding a footer to the web application for credit purposes

**Describe the solution you'd like**
Create a footer component and render inside all working pages

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
1. Footer component created
2. Footer animation and styling

## Purpose
Credit to the developers that worked on this project

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #62

### CODE REVIEW

**Review by Lap343 — DISMISSED** [review]
Nice job with the footer

**Review by Gjhnsn — DISMISSED** [review]
looks good

**Review by krferrell — DISMISSED** [review]
Nice work! Looks great.

File: movie-app/src/scss/components/body/_footer.scss [hunk 1]
```diff
@@ -1,7 +1,6 @@
 .footer {
     width: 100vw;
     height: 300px;
-    // background-color: #9AD5F0;
```
Comment (krferrell) [inline movie-app/src/scss/components/body/_footer.scss]: Good catch

**Review by krferrell — APPROVED** [review]

**Pull request merged**